pom.xml

```diff
@@ -150,9 +144,6 @@
       </developer>
     </developers>
 
-    <!-- Assuming you want to host on @jenkinsci: -->
-    <url>https://wiki.jenkins.io/display/JENKINS/TODO+Plugin</url>
```

Comment:
I would create a Wiki stub. Until posting of docs from GitHub is supported, this is the best way to share info.

Reply:
:+1: